### ISSUE TITLE
Highlights break

### ISSUE THREAD

**Issue by Davidj361** [hexchat/hexchat#1895]
Requirements:
Have a sound for Beep
Have it Beep when you have a highlight
Also the client should blink in the taskbar if you get a highlight

Use case:
1. Right click a channel -> Extra Alerts -> Beep on Message and also Blink on taskbar, have these on.
2. Now disable what you just turned on.
3. The sound should not work, and the taskbar should not blink when you get a highlight.

**Comment by TingPing** [comment]
Yea that is the point, channel options have 3 values, on, off, or unset (use global option).

You just turned them off so this is correct. Sadly a checkbox sucks at displaying 3 states but you can change this with this command: `/chanopt alert_beep unset`. See `/chanopt` for other values.

**Comment by Davidj361** [comment]
Can this be fixed for the GUI so it's more intuitive?

**Comment by TingPing** [comment]
Ideally it would have yet another submenu with a radio selection with 3 options.

**Comment by Davidj361** [comment]
That would be fine by me.

**Comment by Tejeev** [comment]
I agree with Davidj361.  A message in a channel is a different action and priorety from a Highlight in a channel, and should alert as such; differently.

I'm not sure why the sound it plays on both is tied to the beep sound from the settings.  This is a bug, in my opinion.

**Comment by mB-PiBox** [comment]
@Tejeev if you set the "Channel Msg Hilight" (under Chatting>Sounds) to a .wav file of your choice then you would get a different alert sound than the "beep" when a Highlight is received. (note this sound doesn't disable if you uncheck "make a beep sound on.." under Alerts settings

different sounds can also be achieved via scripts. there are a few scripts available floating around, which can be modified.

**Comment by Tejeev** [comment]
@mB-PiBox, I've tried that (first thing I tried actually) and it unfortunately did not work as advertised.  Honestly, I am considering trying something like weechat or irssi as they seem much more versitile and modifiable.  That said, this does not address the OP, so I'm sorry to have taken it off topic.

**Comment by learn-more** [comment]
There are also checkboxes to display tristate:
Off, checked and a 'filled' box to display an inbetween state.

**Comment by zsenseless** [comment]
If you're really set on just having a check-mark or not a check-mark in the Extra Alerts menu, no check-mark should probably indicate unset, because:
1. That's the default behavior when the alerts are initially set without a check-mark. Having no check-mark mean different things in different situations is very unintuitive.
2. Being able to turn Beep on Message on and off for certain channels, given whether or not you're participating in conversation in them, for instance, but still expecting a beep when mentioned seems like the more common expectation. 
3. If you really don't want any beep from a given channel, no matter what, that seems less likely to be something a user would want to toggle on and off very often.
